packages/hardhat-plugin/src/networks.json

```diff
@@ -46,10 +52,39 @@
       "symbol": "Eth",
       "decimals": 18
     },
-    "startBlock": 3530008
+    "startBlock": 3530008,
+    "description": "Ethereum test network. Do not use for production",
+    "isTestNetwork": true
+  },
+  "5": {
+    "unlockAddress": "0x627118a4fB747016911e5cDA82e2E77C531e8206",
+    "multisig": "0x95C06469e557d8645966077891B4aeDe8D55A755",
+    "id": 5,
+    "name": "Goerli (Testnet)",
+    "blockTime": 1000,
+    "subgraphURI": "https://api.thegraph.com/subgraphs/name/unlock-protocol/goerli",
+    "explorer": {
+      "name": "Goerli (Testnet)",
+      "urls": {}
+    },
+    "opensea": {},
+    "requiredConfirmations": 12,
+    "erc20": null,
+    "baseCurrencySymbol": "ETH",
+    "description": "Main Ethereum test network. Do not use for production",
```

Comment:
I'm assuming all these network changes are generated and won't cause any problem?

Reply:
Exactly.